pegen/pegen.h

```diff
@@ -7,6 +7,12 @@
 #include <Python-ast.h>
 #include <pyarena.h>
 
+enum INPUT_MODE {
```

Comment:
If you do `typedef enum  {...} INPUT_MODE;` you can use INPUT_MODE as a type :
```
    INPUT_MODEinput_mode;  // Where the input comes from (0 for file, 1 for string, 2 for fstring)
```

Reply:
Oh, that's what you meant in your other comment. On it!